lib/quickbooks-payments/version.rb

```diff
@@ -1,5 +1,5 @@
 module Quickbooks
   module Payments
-    VERSION = '0.0.5'
+    VERSION = '0.0.6'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.